blog/tests/test_views.py

```diff
@@ -0,0 +1,113 @@
+from django.contrib.auth import login
+from django.test import TestCase
+from django.urls import reverse
+from accounts.models import User
+
+class NonLoginTweetTests(TestCase):
+  def setUp(self):
+    self.tweet_url = reverse('blog:tweet')
+    self.login_url = reverse('accounts:login')
+    self.login_redirect_url = '/accounts/login/?next=/blog/tweet/'

```

Comment:
一例ですが、LOGIN_URLが変更された場合に備えて、以下のようにするといいでしょう。

paramsには、urlencode()を用いましょう

```suggestion
    self.login_redirect_url = f'{reverse(settings.LOGIN_URL)}?{params}'

```